springboot/settings.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remove this file. Nexus configuration should be up to the user and updatableKjar mechanism is optional

Reply:
If we remove this file the user need to change manually the Dockerfile, is it ok ?